src/TickBitmap.sol

```diff
@@ -116,11 +120,11 @@ library TickBitmap {
             // mask = ~((1 << bitPos) - 1)
             // masked = self[wordPos] & mask
             assembly ("memory-safe") {
+                let mask := not(sub(shl(bitPos, 1), 1))
```

Comment:
Why separate this line from `masked := and(...)`?

Reply:
Readability.

Reply:
Define variable close to where they are first used? This is more readable.

Reply:
As you wish.